controllers/apply/get-advice/form.js

```diff
@@ -0,0 +1,398 @@
+                        cy: 'Yr Alban'
+                    }),
+                    value: 'scotland'
+                },
+                {
+                    label: localise({
+                        en: 'Northern Ireland',
+                        cy: 'Gogledd Iwerddon'
+                    }),
+                    value: 'northern-ireland'
+                },
+                {
+                    label: localise({
+                        en: 'Wales',
+                        cy: 'Cymru'
+                    }),
+                    value: 'wales'
+                }
+            ],
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: 'Select a country',
+                        cy: 'Dewiswch wlad'
+                    })
+                }
+            ]
+        }),
+        projectLocation: {
+            name: 'projectLocation',
+            schema: Joi.when('projectCountry', {
+                is: Joi.array().min(2),
+                then: Joi.any().strip(),
+                otherwise: Joi.string().required()
+            }),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: 'Select a location',
+                        cy: 'Dewiswch leoliad'
+                    })
+                }
+            ]
+        },
+        projectLocationDescription: new TextField({
+            label: localise({
+                en: 'Project location',
+                cy: ''
+            }),
+            explanation: localise({
+                en: oneLine`In your own words, describe all of the locations
+                    that you'll be running your project in, e.g.
+                    'Yorkshire' or 'Glasgow, Cardiff and Belfast'`,
+                cy: ``
+            }),
+            name: 'projectLocationDescription',
+            isRequired: false
+        }),
+        projectCosts: {
+            name: 'projectCosts',
+            schema: Joi.friendlyNumber()
+                .integer()
+                .min(10000)
+                .required(),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: 'Enter a total cost for your project',
+                        cy: 'Rhowch gyfanswm cost eich prosiect'
+                    })
+                },
+                {
+                    type: 'number.integer',
+                    message: localise({
+                        en: `Total cost must be a whole number (eg. no decimal point)`,
+                        cy: `Rhaid i’r cost fod yn rif cyflawn (e.e. dim pwynt degol)`
+                    })
+                },
+                {
+                    type: 'number.min',
+                    message: localise({
+                        en: `Must be at least £10,000`,
```

Comment:
In AFA we've made this a constant (`MAX_BUDGET_TOTAL_GBP`) and used that in error messages, despite the extremely unlikely odds of it ever changing. Worth reusing here (or indeed adding a link to AFA for this message)?

Reply:
I have a separate card open for this, so yes will link across to awards for all here.